airbyte-cdk/python/airbyte_cdk/config_observation.py

```diff
@@ -76,7 +84,7 @@ def emit_configuration_as_airbyte_control_message(config: MutableMapping[str, An
     See the airbyte_cdk.sources.message package
     """
     airbyte_message = create_connector_config_control_message(config)
-    print(airbyte_message.model_dump_json(exclude_unset=True))
+    print(orjson.dumps(AirbyteMessageSerializer.dump(airbyte_message)).decode())
```

Comment:
nit: should we move this to a shared function (def print_airbyte_message()) to reduce the risk of messing up the prints? we could share it between sources and destinations

Reply:
We met this construction several times and already have the `AirbyteEntrypoint`.`airbyte_message_to_string()`.

We may met the circular import, and I'd prefer to make the `refactor` in another PR